src/Libraries/Microsoft.Extensions.Caching.Hybrid/Internal/DefaultHybridCache.L2.cs

```diff
@@ -128,6 +213,20 @@ internal void SetL1<T>(string key, CacheItem<T> value, HybridCacheEntryOptions?
         }
     }
 
+    private async ValueTask WritePayloadAsync(string key, CacheItem cacheItem, BufferChunk payload, HybridCacheEntryOptions? options, CancellationToken token)
+    {
+        // bundle a serialized payload inside the wrapper used at the DC layer
+        var maxLength = HybridCachePayload.GetMaxBytes(key, cacheItem.Tags, payload.Length);
```

Comment:
We follow runtime's style - i.e., no `var` unless the type is obvious.
https://github.com/dotnet/extensions/blob/main/CONTRIBUTING.md#dos-and-donts

Applies universally.

Reply:
@RussKie I will do a `var` sweep in a follow up PR - should be a far less daunting one!

Reply:
Sure thing. Thank you